obal/data/playbooks/srpm/metadata.obal.yaml

```diff
@@ -3,3 +3,13 @@ help: >
 
 
   Building source RPMs can be useful when locally building packages.
+variables:
+  build_srpm_dist:
+    parameter: --dist
+    help: Dist to set when building a SRPM
+  build_srpm_scl:
```

Comment:
I thought SRPMs weren't aware of SCL since it's just a combination of source + spec.

Reply:
They technically aren't SCL aware, but tito can still prepend the scl prefix to the front of the SRPM name.